frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -237,17 +329,26 @@ function Application(): JSX.Element {
 						View Traces
 					</Button>
 					<Card>
-						<GraphTitle>Rate (ops/s)</GraphTitle>
-						<GraphContainer>
-							<FullView
-								name="operations_per_sec"
-								fullViewOptions={false}
-								onClickHandler={handleGraphClick('Rate')}
-								widget={operationPerSecWidget}
-								yAxisUnit="ops"
-								onDragSelect={onDragSelect}
-							/>
-						</GraphContainer>
+						{topLevelOperationsIsError && (
+							<Typography>
+								{(topLevelOperationsError as AxiosError).response?.data}
+							</Typography>
+						)}
+						{!topLevelOperationsIsError && !topLevelOperationsLoading && (
```

Comment:
handling this one with enabled

Reply:
Here `enabled` will not work because `isLoading` will only be true when an API request will be sent, enabled will not send the API request in the first place so `isLoading` will be `false` always. I have handled it in another way. Please check the functionality.

Reply:
Okay, now I understood why you wanted to implement `enabled`. Cool added enable in it.